runtime/src/validate_block/tests.rs

```diff
@@ -74,53 +74,38 @@ fn call_validate_block(
 		.map_err(|err| err.into())
 }
 
-fn create_extrinsics() -> Vec<<Block as BlockT>::Extrinsic> {
-	vec![
-		Transfer {
-			from: AccountKeyring::Alice.into(),
-			to: AccountKeyring::Bob.into(),
-			amount: 69,
-			nonce: 0,
-		}
-		.into_signed_tx(),
-		Transfer {
-			from: AccountKeyring::Alice.into(),
-			to: AccountKeyring::Charlie.into(),
-			amount: 100,
-			nonce: 1,
-		}
-		.into_signed_tx(),
-		Transfer {
-			from: AccountKeyring::Bob.into(),
-			to: AccountKeyring::Charlie.into(),
-			amount: 100,
-			nonce: 0,
-		}
-		.into_signed_tx(),
-		Transfer {
-			from: AccountKeyring::Charlie.into(),
-			to: AccountKeyring::Alice.into(),
-			amount: 500,
-			nonce: 0,
-		}
-		.into_signed_tx(),
-	]
-}
-
 fn create_test_client() -> (Client, LongestChain) {
-	TestClientBuilder::new().build_with_longest_chain()
+	TestClientBuilder::new()
+		// NOTE: this allows easier debugging
+		.set_execution_strategy(sc_client_api::ExecutionStrategy::NativeWhenPossible)
+		.build_with_longest_chain()
 }
 
-fn build_block_with_proof(
-	client: &Client,
-	extrinsics: Vec<<Block as BlockT>::Extrinsic>,
-) -> (Block, sp_trie::StorageProof) {
+fn build_block_with_proof(client: &Client) -> (Block, sp_trie::StorageProof) {
 	let block_id = BlockId::Hash(client.info().best_hash);
 	let mut builder = client
 		.new_block_at(&block_id, Default::default(), true)
 		.expect("Initializes new block");
 
-	extrinsics
+	let mut inherent_data = sp_consensus::InherentData::new();
```

Comment:
It would be nice to have a trait that extends `BlockBuilder` to not always have to repeat this code.

Reply:
I tried but I'm struggling with the generics https://github.com/paritytech/cumulus/pull/220/commits/ea5b9e3b31410ec18ad9626b53835ad5d0d0bb60

Logs: https://gitlab.parity.io/parity/cumulus/-/jobs/653559

I think I'm close though!

Reply:
I give up on that one.